app/views/users/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= @user.username %> Profile Page</h1>
```

Comment:
This is going to break in production when you have no users in the database. You should wrap it in a `if @user`